lib/dora-parser/src/error/msg.rs

```diff
@@ -369,22 +369,23 @@ impl Msg {
 
 #[derive(Clone, Debug)]
 pub struct MsgWithPos {
-    pub msg: Msg,
+    pub file: String,
```

Comment:
We should probably make this `Option<String>` or `Option<File>` if we introduce this vector of file names with this PR.

Reply:
I am thinking whether we should move file into `Position`? I think this should probably be our ultimate goal but not sure whether this should (can) be done in this PR.

Please don't feel obliged to solve everything with this single PR. As long as you improve the current status quo it is good enough already (it already is).

FWIW I've once thought about keeping all files in memory for error messages [here](https://github.com/dinfuehr/dora/blob/master/lib/dora-parser/src/lexer/map.rs). But we should definitely not try to tackle that in this PR.